portal_multicurrency_totals/demo/res_partner_demo.xml

```diff
@@ -2,22 +2,12 @@
 <odoo>
 
     <!-- Grant portal access to the customer -->
-    <record id="grant_portal_access" model="portal.wizard.user">
-        <field
-            name="user_id"
-            model="res.partner"
-            eval="[
-                Command.create(
-                    {
-                        'partner_id': ref('base.res_partner_3'),
-                        'email': obj(ref('base.res_partner_3')).email,
-                        'is_portal': True,
-                    }
-                ),
-            ]"
-        />
+    <record id="grant_portal_access" model="portal.wizard">
+        <field name="partner_ids" eval="[Command.set([ref('base.res_partner_3')])]" />
     </record>
-    <function model="portal.wizard.user" name="action_grant_access" eval="[ref('grant_portal_access')]" />
+    <function model="portal.wizard.user" name="action_grant_access">
+        <value model="portal.wizard" eval="obj(ref('grant_portal_access')).user_ids.ids" />
```

Comment:
Since it's only one param, the eval could be included into the `<function>` tag

Reply:
Even if the object is from the portal.wizard model and the function is from the portal.wizard.user model?

Reply:
Good point, I forgot that.